extension/zpagesextension/config.go

```diff
@@ -13,6 +13,10 @@ import (
 // Config has the configuration for the extension enabling the zPages extension.
 type Config struct {
 	confighttp.ServerConfig `mapstructure:",squash"`
+
+	// EnableExpvar indicates whether to enable expvar service.
+	// (default = false)
+	EnableExpvar bool `mapstructure:"enable_expvar"`
```

Comment:
In a potential case we want to enable more things, I would change this to:
```suggestion
	// EnableExpvar indicates whether to enable expvar service.
	// (default = false)
	Expvar ExpvarConfig `mapstructure:"expvar"`
```

And then have:

```
type ExpvarConfig struct {
  Enabled bool `mapstructure: "enabled"`
}
```

Reply:
In the future may allow to configure the path or other things.

Reply:
Thanks for the suggestion, I'll adjust the configurations.